rho/authlistcommand.py

```diff
@@ -50,7 +50,7 @@ def _do_command(self):
         cred_list = vault.load_as_json(utilities.CREDENTIALS_PATH)
 
         for cred in cred_list:
-            if not cred.get('password') == '':
+            if cred.get('password') is not None:
```

Comment:
I think there is a clear difference if a password is provided or not. Just having it as `*********` all the time won't tell the user if he/she has not set. Also it may lead to confusion if a user state it was set because the system as indicating like it was.

I think is better to show empty (None) if not set and the obfuscated version when it is set. The obfuscated version can always be the same length which will make it harder to guess.